assets/css/style.css

```diff
@@ -344,7 +344,7 @@ h1 {
   display: grid;
```

Comment:
Hi Morgan,

I just started looking through this and noticed that the hero CSS changes make the heading text wrap at 414 px and the shadow text is just off the right side of the screen. I realize this is an assignment and not an actual website so it doesn't matter too much. Do you want to leave it or make more adjustments?

![mobile-search-css](https://user-images.githubusercontent.com/93293112/145232608-b80b54d8-80ca-4e44-915e-2eee0b37ed50.PNG)

Reply:
Thanks for catching that -- I think I can fix by adjusting the letter space. I'll update.